src/ui/Tooltip.tsx

```diff
@@ -0,0 +1,27 @@
+import { OverlayTrigger } from 'react-bootstrap';
+import { Tooltip as BootstrapTooltip } from 'react-bootstrap';
+
+import { QuestionCircleIcon } from '@/pages/NewFeatureTooltip';
+
+type TooltipPlacement = 'top' | 'right' | 'bottom' | 'left';
```

Comment:
Would maybe make `Tooltip`'s `Props` extend `OverlayProps` and just do `{...props}` on it, then we can use any of the placements and have it synced with the package